server/routes/index.js

```diff
@@ -282,6 +285,17 @@ var routes = {
       metadata.glassroomnyc = true;
     }
 
+    if (!customerId) {
+      request.log(['error', 'stripe', 'recurring'], {
+        request_id,
+        err: 'Customer ID not present in the cookie'
+      });
+
+      return reply(boom.badRequest('An error occurred while fetching the customer for this monthly upsell'));
```

Comment:
can we make this error ''An error occurred while creating this monthly donation"